dev/breeze/src/airflow_breeze/utils/publish_docs_helpers.py

```diff
@@ -31,6 +31,22 @@
 ROOT_DIR = Path(__file__).parents[5].resolve()
 PROVIDER_DATA_SCHEMA_PATH = ROOT_DIR / "airflow" / "provider.yaml.schema.json"
 
+providers_prefix = "apache-airflow-providers-"
+
+
+def get_provider_name_from_short_hand(short_form_providers: tuple[str]):
```

Comment:
Should this method be also used in `doc_build_params.py`?

Reply:
You are right. I repeated the method in both places, let me refactor so that I can use it from a common place.

Reply:
Moving the helper function to a general utils file and then reusing it everywhere